src/index.html

```diff
@@ -8,7 +8,7 @@
       href="./styles/main.scss"
     />
   </head>
-  <body>
+  <body id="tree">
```

Comment:
The link tag is referencing a SCSS file (`main.scss`). Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Ensure that the SCSS file is compiled to a CSS file and update the `href` attribute to point to the resulting CSS file.